util/instance.cc

```diff
@@ -154,7 +154,7 @@ bool Instance::init_ssl()
     _last_error= message.str();
     return false;
   }
-#endif // defined(HAVE_CYASSL) && HAVE_CYASSL
+#endif // defined(HAVE_WOLFSSL) && HAVE_WOLFSSL
```

Comment:
So, this is just a search and replace change. Once we have WolfSSL in place, we can think about fixing bugs.